src/registrar/tests/test_models.py

```diff
@@ -2131,13 +2128,6 @@ def test_is_purpose_complete(self):
         self.domain_request.save()
         self.assertFalse(self.domain_request._is_purpose_complete())
 
-    @less_console_noise_decorator
-    def test_is_submitter_complete(self):
```

Comment:
Removed test for submitter. Did not replace with an equivalent for creator since creator, as a required field, cannot be set to null